app/models/form/closed-question.js

```diff
@@ -22,7 +22,7 @@ export default class ClosedQuestion extends Model {
   }
 
   get sortedOptions() {
-    return this.options.toArray().sort((a) => a.position);
+    return this.options.sortBy('position');
```

Comment:
Isn't that a JQuery function?

Reply:
ehm, I don't know? is it? https://api.emberjs.com/ember/release/classes/EmberArray shows it as a normal method for ember